simpl/server.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2013-2015 Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Bottle Server Module."""
+
+import logging
+import os
+
+import bottle
+
+LOG = logging.getLogger(__name__)
+
+
+class EventletLogFilter(object):  # pylint: disable=R0903
```

Comment:
You raise a valid question which we should bring up for the team to discuss.  Most of our other projects run pylint in the tox runs. I would vote to include it (with a customized pylintrc file). I've raised #55 to host that discussion.

WRT to this PR, pylint comments are in many other places in this codebase (as well as in many of our other projects - most of which actually include pylint in the tox runs) - as well as a pylintrc file - so this line does not diverge from what we've been doing.

It does document that we've intentionally not included public methods in this class. And it does  it in a way that is compatible with one of the most commonly used linting tools for python.

While removing this line is simple, I'd rather stay consistent with the rest of the codebase and remove it if we choose to not use pylint and remove pylint comments from the whole code base.